releng/test_config.yaml

```diff
@@ -522,21 +522,21 @@ k8sVersions:
     - --extract=ci/latest
     version: master
   beta:
+    args:
+    - --extract=ci/latest
+    version: master
```

Comment:
I don't think updating this makes sense until we have `ci/latest-1.20`

Reply:
@spiffxp -- Not quite. This is vestigial from when release branches were cut during the beta.0.

Now that we cut branches at rc.0 and cut betas from `master`, this is correct.
[v1.20.0-beta.1](https://github.com/kubernetes/kubernetes/releases/tag/v1.20.0-beta.1) was just released on Tuesday, so the "beta" jobs are essentially behind a version.

This was the motivation for introducing a `stable4` marker in https://github.com/kubernetes/test-infra/pull/18169:

> * Drop test config for beta jobs, add stable4 jobs
>   beta markers shift depending on the stage in the release cycle and can
>   end up being inaccurate without consistent care and feeding of the
>   generated jobs.
>   Here we remove the beta job configs (not the k8s-beta version marker;
>   that'll happen in an upcoming commit) and establish a new `stable4`
>   version config.
>   This serves a few purposes:
>   
>   * beta releases are now cut from the main (`master`) branch, so the
>     `dev` `k8sVersions`/`nodeK8sVersions` configs are now sufficient to
>     describe the beta phase of the release cycle
>   * now that we've expanded the support cycle for Kubernetes versions
>     (from 1.19 on) to 12+ months, removing the previous jobs no longer
>     falls neatly in line with the timing of the next branch cut. The
>     `stable4` job configs will now allow us to maintain generated jobs
>     through the newly extended support cycle.